src/screens/Form/index.tsx

```diff
@@ -1,57 +1,92 @@
 import { Card } from 'components';
-import React, { useState } from 'react';
-import { KeyboardAvoidingView, Platform, ScrollView, Text } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import { Alert, KeyboardAvoidingView, Platform, ScrollView, Text } from 'react-native';
 import { GestureHandlerRootView, TouchableNativeFeedback } from 'react-native-gesture-handler';
 import { theme } from 'utils/theme';
 import { DatePicker } from '../../components/DatePickerApp';
 import { FormHeader } from '../../components/FormHeader';
 import { Input } from '../../components/Input';
 import { useNavigation } from '@react-navigation/native';
+import { makeViagemEntradaService, makeViagemService } from 'core';
+import { ViagemModel } from 'core/database/models';
 
-const data = [
-  {
-    id: 1,
-    local: 'Pitangueiras',
-    data: new Date('2022-01-01'),
-    descricao:
-      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Ratione aut commodi optio. Nostrum, numquam magni.',
-  },
-  {
-    id: 2,
-    local: 'Pitangueiras',
-    data: new Date('2022-01-01'),
-    descricao:
-      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Ratione aut commodi optio. Nostrum, numquam magni.',
-  },
-  {
-    id: 3,
-    local: 'Pitangueiras',
-    data: new Date('2022-01-01'),
-    descricao:
-      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Ratione aut commodi optio. Nostrum, numquam magni.',
-  },
-  {
-    id: 4,
-    local: 'Pitangueiras',
-    data: new Date('2022-01-01'),
-    descricao:
-      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Ratione aut commodi optio. Nostrum, numquam magni.',
-  },
-];
+const viagemService = makeViagemService();
+const viagemEntradaService = makeViagemEntradaService();
 
 export function Form({ route }) {
   const navigation = useNavigation();
   const { params } = route;
   const { apenasConsulta, title } = params;
-
+  const [dateValue, setDateValue] = useState<Date>();
```

Comment:
Iniciar o estado com a data atual

```suggestion
  const [dateValue, setDateValue] = useState<Date>(new Date());
```